### ISSUE TITLE
[Icon Request] Slack and Discord

### ISSUE THREAD

**Issue by OpsTalkJordan** [EliverLara/candy-icons#72]
- [ ] [Slack](https://brandfolder.com/slack/logos)
- [ ] [Discord](https://discordapp.com/branding)

The above links will take you directly to the official branding sites with .svg and .png icon downloads. I hope that makes it a bit easier.

**Comment by EliverLara** [comment]
Hi, there are already icons for those applications, plase make sure you have not installed them via snap.

**Comment by OpsTalkJordan** [comment]
Both have been installed via RPMs (Fedora 31 Gnome). I'll try to figure out why I'm not seeing the right ones. Thanks!

**Comment by OpsTalkJordan** [comment]
Figured it out. By default the `slack.desktop` and `discord.desktop` files found in `/usr/share/applications/` have absolute paths for their icons key-value pair while the apps that are showing correct icons have just an application name. Changing the value of the Icon key to match the name found in `~/.local/share/icons/candy-icons/apps/scalable` fixed it.

For anyone else that runs into this I installed the Discord RPM from RPM Fusion and the Slack app directly from the official Downloads site. 

Old:
```
❯ cat /usr/share/applications/discord.desktop 
[Desktop Entry]
Name=Discord
StartupWMClass=discord
Comment=All-in-one voice and text chat for gamers that's free, secure, and works on both your desktop and phone.
GenericName=Internet Messenger
Exec=/usr/bin/Discord
Icon=/usr/lib64/discord/discord.png
Type=Application
Categories=Network;InstantMessaging;
X-Desktop-File-Install-Version=0.23
```

New:
❯ cat /usr/share/applications/discord.desktop 
[Desktop Entry]
Name=Discord
StartupWMClass=discord
Comment=All-in-one voice and text chat for gamers that's free, secure, and works on both your desktop and phone.
GenericName=Internet Messenger
Exec=/usr/bin/Discord
Icon=discord
Type=Application
Categories=Network;InstantMessaging;
X-Desktop-File-Install-Version=0.23
```
